tests/test.h

```diff
@@ -83,6 +83,20 @@ void add_unordered_string_output(char* output);
  */
 void check_inputs(uint64_t expected_buttons, float expected_joysticks[4], robot_desc_field_t source);
 
+/******************** UDP Device Data Checks ******************/
+
+/**
+ * Checks that the given device was returned by the UDP thread.
```

Comment:
Wdym by returned by the UDP thread? All device data are sent over UDP right?

Reply:
I meant sent by the UDP thread. I will fix it